fl4health/clients/mr_mtl_client.py

```diff
@@ -83,8 +83,15 @@ def get_parameters(self, config: Config) -> NDArrays:
         Returns:
             NDArrays: LOCAL model weights to be sent to the server for aggregation
         """
-        assert self.model is not None and self.parameter_exchanger is not None
-        return self.parameter_exchanger.push_parameters(self.model, config=config)
+        if not self.initialized:
+            # If initialized==False, we are doing client side initialization (get_parameters called before fit)
+            # so we must call setup_client first
+            self.setup_client(config)
+            # Need all parameters even if normally exchanging partial
+            return FullParameterExchanger().push_parameters(self.model, config=config)
+        else:
+            assert self.model is not None and self.parameter_exchanger is not None
+            return self.parameter_exchanger.push_parameters(self.model, config=config)
```

Comment:
Maybe this is a naive question, but do we actually need to override the get_parameters function for MR-MTL clients? It looks identical to that of BasicClient.

Reply:
I thought the same thing but I figured there was a reason haha, I will just call basic one

Reply:
I think you can just drop this override 🙂 and it will be called.